src/IbuMethods.cpp

```diff
@@ -18,7 +18,7 @@
  */
 
 #include "IbuMethods.h"
-#include <cmath>
```

Comment:
The reason is quite simple. The last time i made a pull request you asked me to avoid stl functions, so when i saw that there were a QMath i decided to do some refactoring and switch to it. I then continued and forgot about it while i did the rest. I must say that i didn't actually look in the documentation for QMath since verything compiled and worked. I must admitt that it feels somewhat embarrassing that i didn't look in the documentation before making the change.